src/components/Export.tsx

```diff
@@ -445,7 +445,7 @@ export default function Export(inputProps: ExportButtonProps): ReactElement {
         centered={true}
         // Don't allow cancellation of modal by clicking off it when the recording is happening
         maskClosable={!isRecording}
-        getContainer={modalContextRef.current || undefined}
+        getContainer={modalContainerRef || undefined}
         footer={modalFooter}
       >
```

Comment:
Would it make any sense at all to create a shared `Modal` wrapper that captures and uses this context, rather than having to remember to do this for every current and future `Modal`? Absolutely valid to say "no" given how many more modals we expect to use (or not) and/or how much more complicated modals may (or may not) get.

Reply:
Yeah, that makes sense, I'll take a stab at this and see if I can get it to work as expected. We'll still have to do it manually for any use of the static API (see #289).

Reply:
Fixed! I can't find a way to elegantly do this for the static modal calls, though I'd like to. I could provide styled versions of them (`styledStaticModal.confirm`, etc.) but I wouldn't have a way to access the ref since they don't exist inside a React component and I therefore can't use `useContext`... I've seen a few places where people assign an `id` to the wrapper container, but I'm not sure if that's a robust solution.

https://github.com/allen-cell-animated/nucmorph-colorizer/pull/280/files#diff-52e0fc4234b72537ec0c614472063a5c4b962ea1cffc66fd8a92e69616d0d571R215-R227